### PR TITLE
WRKLDS-1016: bump k8s to 1.29.1

### PR DESCRIPTION
**Pull request by atiratree**

### CODE REVIEW

File: pkg/cmd/controller/standalone_apiserver.go [hunk 1]
```diff
@@ -47,7 +48,7 @@ func RunControllerServer(servingInfo configv1.HTTPServingInfo, kubeExternal clie
 		return err
 	}
 	sarClient := kubeExternal.AuthorizationV1()
-	remoteAuthz, err := authzwebhook.NewFromInterface(sarClient, 5*time.Minute, 5*time.Minute, *authzwebhook.DefaultRetryBackoff(), authzwebhook.AuthorizerMetrics{
+	remoteAuthz, err := authzwebhook.NewFromInterface(sarClient, 5*time.Minute, 5*time.Minute, *authzwebhook.DefaultRetryBackoff(), authorizer.DecisionNoOpinion, authzwebhook.AuthorizerMetrics{
```
Comment (atiratree) [inline pkg/cmd/controller/standalone_apiserver.go:51]: using the original default behaviour https://github.com/kubernetes/kubernetes/commit/2e2f51a4417d93b5505091d28b319365dc95e137#diff-8b41b5e2c7d3bb27fa452e58ab0a4120b02d3a7a16d8961954f5c3c270ab8935L125

File: pkg/build/controller/strategy/util.go [hunk 2]
```diff
@@ -114,28 +113,29 @@ func setupDockerSocket(pod *corev1.Pod) {
 // mountConfigMapVolume is a helper method responsible for actual mounting configMap
 // volumes into a pod.
 func mountConfigMapVolume(pod *corev1.Pod, container *corev1.Container, configMapName, mountPath, volumeSuffix string, volumeSource *corev1.VolumeSource) {
-	mountVolume(pod, container, configMapName, mountPath, volumeSuffix, policy.ConfigMap, volumeSource)
+	mountVolume(pod, container, configMapName, mountPath, volumeSuffix, "configMap", volumeSource)
```
Comment (atiratree) [inline pkg/build/controller/strategy/util.go:116]: removed in https://github.com/kubernetes/kubernetes/pull/119806

**Review by soltysh — APPROVED** [review]
/approve

**Pull request merged**